engine/src/flutter/runtime/runtime_delegate.h

```diff
@@ -32,6 +32,8 @@ class RuntimeDelegate {
                       std::unique_ptr<flutter::LayerTree> layer_tree,
                       float device_pixel_ratio) = 0;
 
+  virtual void ResizeView(int64_t view_id, double width, double height) = 0;
```

Comment:
Consider returning `bool` instead to indicate whether the platform could actually resize the view. The default could just return `false`. Then the runtime controller can log when the result is `false` saying something like "Could not resize platform view.". You can do thread checks on platforms that don't support off main thread resizes and the threads haven't been merged.

Reply:
I like keeping the thread checks in the respective platform implementations.

Reply:
If we merge the resize view callback with the present view callback, `FlutterCompositor.present_view_callback`, the engine should assume the resize operation failed if the present view callback returns `false`.